resources/qml/dialogs/ImagePackDeleteDialog.qml

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import Qt.labs.platform 1.1 as P
+import QtQuick
+import im.nheko
+
+P.MessageDialog {
+    id: deleteStickerPackRoot
+
+    property SingleImagePackModel imagePack
+    property ImagePackListModel packList
+
+    text: "This will delete the sticker pack: " + imagePack.packname
```

Comment:
I'd suggest removing `informativeText` and setting `text` to `"Are you sure you want to delete \"" + imagePack.packname) + "\"?"`. The current message feels needlessly verbose.